packages/next/build/webpack-config.ts

```diff
@@ -897,6 +897,7 @@ export default async function getBaseWebpackConfig(
     hasSupportCss: !!config.experimental.css,
     hasSupportScss: !!config.experimental.scss,
     assetPrefix: config.assetPrefix || '',
+    scssIncludePaths: config.experimental.scssIncludePaths,
```

Comment:
I think we should make it something like:

```
config.experimental.scss.includePaths
```

So nesting it under `scss`

Reply:
We should also support the full `sassOptions` object instead of just `includePaths` (check the issue description in NEXT-121 for context as to why)